### DISCUSSION TITLE
시맨틱 태그의 사용과 navbar에 대하여

### DISCUSSION THREAD

**Discussion by hozzijeong** [woowacourse-teams/2023-pium#151] [질문있어요] [answered]
저희가 작성한 `Root` 파일을 보면 모든 페이지를 `main`태그로 감싸고 있습니다. 하나의 페이지에 `main`태그가 유일해야 한다는 생각 아래 해당 행위는 올바른 코드라고 생각합니다.

하지만 페이지에따른 navbar의 위치가 어디가 맞는가? 라는데 의문이 생겼습니다. `main`태그에 들어가야 할 내용을 보면 다음과 같습니다

> 'main' 요소의 콘텐츠는 문서의 유일한 내용이어야 합니다. 사이드바, 탐색 링크, 저작권 정보, 사이트 로고, 검색 폼 등 여러 문서에 걸쳐 반복되는 콘텐츠는 포함해선 안됩니다. 그러나 검색 폼이 페이지의 주요 기능이라면 예외로 둘 수 있습니다. -MDN-

여기서 navbar를 여러 문서에 걸쳐 반복되는 콘텐츠로 볼 것인가? 에 대한 여러분들의 생각이 궁금합니다. 저도 좀 애매한것 같긴 해서요

**Comment by bassyu** [comment, marked as answer]
클린 말대로 페이지에서 nav와 main이 분리되는 것이 좋겠네요!
제가 의도한 Root는 max-width를 통해서 페이지 하나의 최대 너비를 고정하는 것이었는데요,
오히려 Root의 main 태그를 걷어내고 페이지마다 nav 와 main을 적절하게 조합하는 방향으로 가면 좋을 것 같습니다!
